### ISSUE TITLE
Build baseline langchain Agents

### ISSUE THREAD

**Issue by LanesGood** [wri/project-zeno#1]
### Proposed Activity
Build the baseline GFW agents that can query data based on prompts.

- [x] #2
- [x] #3
- [ ] #4
- [ ] #5
- [x] #10

### Context

Following the Discovery Workshop and our initial understanding of integration needs, we plan to set up GFW agents in Langchain as baseline proof of concept. These will be built independently as single-use tools which can be enhanced and improved in parallel to later use case applications.

**Comment by yellowcap** [comment]
We made good progress in first agents. Closing this in favor of first user story.
